docs/api/README.md

```diff
@@ -1,6 +1,6 @@
-# CRDs
+# API reference for CRDs
```

Comment:
Should this be simply "API reference" as it was before?

Reply:
I think what you did here is better.